### ISSUE TITLE
Carbonite-specific issue

### ISSUE THREAD

**Issue by josephdadams** [josephdadams/TallyArbiter#92]
"We’ve found a small bug in the code since we’ve built hardware around your code. If you edit the destination within a device, say from ME1 to ME2 it changes the destination for all devices. We’ve found a workaround in that if you delete the patch within device and then create new it’s ok but would be good to get a fix when you next dive into the code"

**Comment by MatteoGheza** [comment]
@josephdadams what can we do for this issue? Is this working in new versions?

**Comment by hrueger** [comment]
Will probably be fixed in #204

**Comment by JTF4** [comment]
I revisited the carbonite source type and it looks like this issue was only half fixed. If you edit one or add a new one, the bus settings on that device will overwrite all the other devices. 

The carbonite source type is also just broken in general. The moment you start specifying different MEs, the module's processing falls apart.

**Comment by josephdadams** [comment]
Yes part of it is UI based - we need to present the user a list of bus options available to that specific source type when assigning the address. For example, address 1 on MiniME4, or Aux 5, etc.

**Comment by JTF4** [comment]
![image](https://user-images.githubusercontent.com/63563936/153082462-e38578ee-32aa-43a7-a021-6af70c4b2dfc.png)
This is what we currently have. It has the right options, but those options do not get communicated and saved properly.

**Comment by josephdadams** [comment]
My mistake, I thought it was not implemented in the UI in v3.
